roles/openshift_logging_mux/defaults/main.yml

```diff
@@ -1,7 +1,7 @@
 ---
 ### General logging settings
-openshift_logging_mux_image_prefix: "{{ openshift_hosted_logging_deployer_prefix | default('docker.io/openshift/origin-') }}"
-openshift_logging_mux_image_version: "{{ openshift_hosted_logging_deployer_version | default('latest') }}"
+openshift_logging_mux_image_prefix: "{{ openshift_logging_image_prefix | default(__openshift_logging_image_prefix) }}"
```

Comment:
I realize that you did not make this change here, but why do we have a mux image prefix and version separate from the fluentd image?